modules/basic-skills/src/backend/index.ts

```diff
@@ -52,7 +52,7 @@ const entryPoint: sdk.ModuleEntryPoint = {
   translations: { en, fr, es },
   definition: {
     name: 'basic-skills',
-    menuIcon: 'fiber_smart_record',
+    menuIcon: 'build',
```

Comment:
Same for the basic-skills module. In fact, every module that has `noInterface: true` does not require a menu icon!